uitools/README.md

```diff
@@ -112,18 +112,13 @@ get added to QtCreator when you install the ArcGIS Runime SDK for Qt.
 
 - In QtCreator choose `File/New file or project/ArcGIS/ArcGIS Maps 200.4.0 Qt Quick C++ App`
 - choose settings to match the platform etc. you are building for
-- in the `CMakeLists.txt` file of your new app, add this line to get access to the `setup_toolkit` function:
-
-```CMake
-include(path/to/toolkitcpp.cmake)
-```
-- in the `CmakeLists.txt` file of your new app, call `setup_toolkit` after you've defined your target and pass in your target name and the path to the toolkit
-```CMake
-qt_add_executable(target_name ${SOURCE_FILES})
-...
-setup_toolkit(target_name path_to_toolkit)
-```
-
+- In the `CMakeLists.txt` file of your new application, you need to make two changes:
+  - Uncomment the `include` line below, and replace `path_to_toolkitcpp.cmake` with the actual path to your `toolkitcpp.cmake` file.
+  - Uncomment the `setup_toolkit` line below, and replace `target_name` with the name of your target, and `path_to_toolkit` with the path to your toolkit directory.
+  ```CMake
+  # include(path_to_toolkitcpp.cmake)
+  # setup_toolkit(target_name path_to_toolkit)
+  ```
```

Comment:
I would tweak this so that `path_to` is not part of the filename. It should be like this:
```cmake
include(/path/to/toolkitcpp.cmake)
```
Or something like that. When I first saw this it made me wonder for a moment if we had a filed called `path_to_toolkit` somewhere.

Reply:
Good point, thanks.